tree-construction/template.dat

```diff
@@ -1625,3 +1625,57 @@ template
 |         name="q"
 |     <div>
 |       "second"
+
+#data
+<!DOCTYPE HTML><template><tr><td>cell</td></tr></template>
+#document-fragment
+template
+#errors
+#document
+| <!DOCTYPE html>
+| <html>
+|   <head>
+|     <template>
+|       content
+|         <tr>
+|           <td>
+|             "cell"
+|   <body>
+
+#data
+<!DOCTYPE HTML><template> <tr> <td>cell</td> </tr> </template>
+#document-fragment
+template
+#errors
+#document
+| <!DOCTYPE html>
+| <html>
+|   <head>
+|     <template>
+|       content
+|         " "
+|         <tr>
+|           " "
+|           <td>
+|             "cell"
+|           " "
+|         " "
+|   <body>
+
+#data
+<!DOCTYPE HTML><template><tr><td>cell</td></tr>a</template>
+#document-fragment
+template
+#errors
+(1,48): foster-parenting-character
```

Comment:
The parse error happens when handling the `</template>` token: "Anything else" in https://html.spec.whatwg.org/multipage/parsing.html#parsing-main-intabletext

```suggestion
(1,59): foster-parenting-character
```